GliaWidgets/View/Chat/Message/VisitorChatMessageStyle.swift

```diff
@@ -1,10 +1,25 @@
 import UIKit
 
+/// Style of a visitor's message.
 public class VisitorChatMessageStyle: ChatMessageStyle {
+    /// Font of the message's status text.
```

Comment:
I haven't marked it above, but the more times I read it, the more I noticed that here `'s` is also not needed. It is the status of a message, not the one that belongs to it. This comment applies to almost all places where `message's` is encountered.